api/src/main/java/io/minio/BucketRegionCache.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Minio Java SDK for Amazon S3 Compatible Cloud Storage, (C) 2015 Minio, Inc.
+ *MinIO Java SDK for Amazon S3 Compatible Cloud Storage, (C) 2015MinIO, Inc.
```

Comment:
same here, seems like in some places space in front of Minio is trimmed.

Reply:
Fixed